bindings/java/impl/src/main/com/apple/foundationdb/FDBDatabase.java

```diff
@@ -26,6 +26,7 @@
 import java.util.concurrent.atomic.AtomicReference;
 import java.util.function.Function;
 
+import com.foundationdb.*;
```

Comment:
Does this repo allow `*` imports? I'm not actually sure (but generally have seen people prefer explicit imports for each imported class).